keps/sig-instrumentation/1602-structured-logging/README.md

```diff
@@ -568,7 +568,7 @@ Items marked with (R) are required *prior to targeting to a milestone / release*
 - [x] (R) Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
 - [x] (R) Graduation criteria is in place
 - [x] (R) Production readiness review completed
-- [ ] Production readiness review approved
+- [x] Production readiness review approved
```

Comment:
Github makes it impossible to add comments to lines that you didn't change - it's such a stupid thing....

So I will be adding comments here - please apply them:
590: describe for which components that flag is (I'm assuming all, right?)
590 and further: nit: please add answers in separate lines - it's much easier to read it then

Reply:
Done.